### PR TITLE
Add a FS api endpoint

### PR DESCRIPTION
**Pull request by Sumukh**
TODO: 
- [x] Fix typo in API name
- [x] Resolve merge conflict 
- [ ] Add info to documentation 
- [ ] Add tests

### CODE REVIEW

File: server/controllers/api.py [hunk 1]
```diff
@@ -463,7 +463,7 @@ def get(self, user, aid, email):
                 'has_more':  has_more}
         return data
 
-class ExportFinal(Resource):
+class ExportSubmsissions(Resource):
```
Comment (Sumukh) [inline server/controllers/api.py]: Spelling. Should be `ExportSubmissions`

File: server/controllers/api.py [hunk 2]
```diff
@@ -563,8 +593,9 @@ def get(self, name=None):
 
 api.add_resource(V3Info, '/v3/')
 api.add_resource(Backup, '/v3/backups/', '/v3/backups/<string:key>/')
+api.add_resource(ExportSubmsissions, '/v3/assignment/<int:aid>/submissions/')
```
Comment (Sumukh) [inline server/controllers/api.py]: See spelling comment above.

**Comment by the-zebulan** [comment]
@Sumukh ,

I made the changes you asked for (fixed spelling mistakes including a few others I found) but I'm not sure if I did the PR correctly. I wasn't sure how (or if it was possible) to add to this PR, so I made a different one, #928 .

**Comment by Sumukh** [comment]
Implemented in #928. Still needs tests, but I'm going to close this PR

**Pull request closed without merging**